route53.tf

```diff
@@ -0,0 +1,29 @@
+# ------------------------------------------------------------------------------
+# DNS records that support the CloudFront endpoints and application.
+# ------------------------------------------------------------------------------
+
+resource "aws_route53_record" "rules_vm_A" {
+  provider = aws.route53resourcechange
+
+  alias {
+    evaluate_target_health = false
+    name                   = aws_cloudfront_distribution.rules_s3_distribution.domain_name
+    zone_id                = "Z2FDTNDATAQYW2"
```

Comment:
Is there any way to avoid hard-coding these Route53 zone IDs?

Reply:
Not that I know of, but please take a look at https://github.com/cisagov/publish-egress-ip-terraform/pull/6/commits/5f5135fa2615c1456463493b76eaaf5fa5e882c5 and let me know if you think that helps.

Reply:
Ah, that is much better.  Thank you.